Tasks/MavenV3/task.loc.json

```diff
@@ -86,7 +86,7 @@
       "name": "testResultsFiles",
       "type": "filePath",
       "label": "ms-resource:loc.input.label.testResultsFiles",
-      "defaultValue": "**/TEST-*.xml",
+      "defaultValue": "**/surefire-reports/Test-*.xml",
```

Comment:
Need to be `TEST-*.xml`, on Linux it will be case sensitive.

Reply:
http://maven.apache.org/surefire/maven-surefire-plugin/index.html

By default, these files are generated in ${basedir}/target/surefire-reports/TEST-*.xml.